### ISSUE TITLE
Limit the number of rings displayed at any one time

### ISSUE THREAD

**Issue by mph070770** [vasturiano/sunburst-chart#16]
I have a limited resolution and size of display, so it would be really useful to be able to define the number of rings that are displayed. For example, the current ring and the next ring of parameters.
Thanks

**Comment by stefanseefeld** [comment]
I agree. Limiting the depth of the chart (perhaps even interactively !) would be a useful feature, not just in the context of low-resolution displays.
Note that the sunburst widget used in the `snakeviz` tool does just that: https://jiffyclub.github.io/snakeviz/#controls.
